MAUI/NavigationDrawer/Side-Pane-Content.md

```diff
@@ -0,0 +1,363 @@
+
+using Syncfusion.Maui.NavigationDrawer;
+
+namespace NavigationSample;
+
+public partial class NavigationDrawerView : ContentPage
+{
+	public NavigationDrawerView()
+	{
+		InitializeComponent();
+		navigationDrawer.DrawerSettings.DrawerWidth = 250;
+	}
+
+    void hamburgerButton_Clicked(object sender, EventArgs e)
+    {
+        navigationDrawer.ToggleDrawer();
+    }
+}
+  
+{% endhighlight %}
+
+{% endtabs %}
+
+![Header]()
+
+## Header Height
+
+The height of the drawer header content can be adjusted using the [`DrawerHeaderHeight`]() property.
+
+N> [`DrawerHeaderView`]() will be disabled by setting the [`DrawerHeaderHeight`]() to zero.
+
+{% tabs %}
+
+{% highlight xaml %}
+    
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.DrawerSettings>
+        <navigationdrawer:DrawerSettings DrawerHeaderHeight="150">
+            <navigationdrawer:DrawerSettings.DrawerHeaderView>
+                <Grid BackgroundColor="#1aa1d6">
+                    <VerticalStackLayout VerticalOptions="Center"
+                                         HorizontalOptions="Center">
+                        <Label Text="Header View"/>
+                    </VerticalStackLayout>
+                </Grid>
+            </navigationdrawer:DrawerSettings.DrawerHeaderView>
+        </navigationdrawer:DrawerSettings>
+    </navigationdrawer:SfNavigationDrawer.DrawerSettings>
+    <navigationdrawer:SfNavigationDrawer.ContentView>
+        <VerticalStackLayout>
+            <HorizontalStackLayout BackgroundColor="#1aa1d6" 
+                  HeightRequest="50" 
+                  VerticalOptions="Start"
+                  Padding="5,0,0,0">
+                <ImageButton x:Name="hamburgerButton"
+                             HeightRequest="50"
+                             WidthRequest="50"
+                             HorizontalOptions="Start"
+                             Source="hamburgericon.png"
+                             BackgroundColor="#1aa1d6"
+                             Clicked="hamburgerButton_Clicked"/>
+            </HorizontalStackLayout>
+        </VerticalStackLayout>
+    </navigationdrawer:SfNavigationDrawer.ContentView>
+</navigationdrawer:SfNavigationDrawer>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.NavigationDrawer;
+
+namespace NavigationSample;
+
+public partial class NavigationDrawerView : ContentPage
+{
+	public NavigationDrawerView()
+	{
+		InitializeComponent();
+        navigationDrawer.DrawerSettings.DrawerWidth = 250;
+        navigationDrawer.DrawerSettings.DrawerHeaderHeight = 150;
+	}
+
+    void hamburgerButton_Clicked(object sender, EventArgs e)
+    {
+        navigationDrawer.ToggleDrawer();
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Header height]()
+
+## Footer Content
+
+As the name suggests it is displayed at the bottom of the drawer. The  [`DrawerFooterView`]() property is used to set the footer content of the drawer. 
```

Comment:
As the name suggests it is displayed at the bottom of the drawer. The `DrawerFooterView` property is used to set the footer content of the drawer.

Reply:
Changed